typescript/agentkit/README.md

```diff
@@ -469,6 +469,24 @@ const walletProvider = await CdpWalletProvider.configureWithWallet({
 });
 ```
 
+#### Configuring gas parameters
```

Comment:
Let's add a link to this section in the table of contents https://github.com/coinbase/agentkit/blob/main/typescript/agentkit/README.md#table-of-contents